scripts/run_backend_tests.py

```diff
@@ -659,6 +691,44 @@ def check_coverage(
     process = subprocess.run(
         cmd, capture_output=True, encoding='utf-8', env=env,
         check=False)
+
+    coverage_output_lines = process.stdout.split('\n')
+    # The `process.stdout` is a string containing the coverage report,
+    # structured as follows:
+    # process_stdout_structure =
+    # Name       Stmts Miss Branch BrPart Cover Missing.
+    # -------------------------------------------------.
+    # file1.py      10    0      2      0   100%   12, 14.
+    # file2.py      20    0      4      0    90%   32, 34, 36, 38.
+    # -------------------------------------------------
+    # TOTAL        30    0      6      0   95%.
+    incomplete_coverage_lines: List[str] = []
+    # Lines with incomplete coverage are appenede to incomplete_coverage_lines.
```

Comment:
Spelling: appended

Reply:
@seanlip i have made this change in new commit. But waiting for your reply for [this](https://github.com/oppia/oppia/pull/19886#discussion_r1527197909) comment . so that I can make commit having all the changes.